### PR TITLE
nginx: add http_sub_module

### PR DESCRIPTION
**Pull request by Ansuel**
Someone requested this module to be included, so create the config flag for it.
@hnyman 

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

File: net/nginx/files-luci-support/luci_nginx.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-user  root;
+user nobody nogroup;
```
Comment (yousong) [inline net/nginx/files-luci-support/luci_nginx.conf]: separate commit please

Comment (Ansuel) [reply net/nginx/files-luci-support/luci_nginx.conf]: didn't want to create 2 commit for that small change but if it's a problem i will split them then...

Comment (yousong) [reply net/nginx/files-luci-support/luci_nginx.conf]: It's okay to have multiple commits in single pull request

**Comment by Ansuel** [comment]
@hnyman can you merge this?

**Pull request merged**